kubetest2-gke/deployer/up.go

```diff
@@ -78,6 +79,7 @@ func (d *deployer) Up() error {
 					args = append(args, fmt.Sprintf("--workload-pool=%s.svc.id.goog", project))
 				}
 				args = append(args, subNetworkArgs...)
+				args = append(args, privateClusterArgs...)
```

Comment:
+1 this is the pattern i'd want in the future too!
i.e. each set of options will return the args that we pass to the create call